frontend/packages/rhoas-plugin/src/components/service-list/ServiceInstance.tsx

```diff
@@ -39,28 +52,48 @@ const ServiceInstance: React.FC<ServiceInstanceProps> = ({
 
   const { t } = useTranslation();
 
+  const noKafkaInstancesExist = (
+    <Trans t={t} ns="rhoas-plugin">
+      To make sure the instance exists and that you&lsquo;re authorized to access it, you can see
+      your Kafka instances at{' '}
+      <a href="https://cloud.redhat.com/openshift/token" rel="noopener noreferrer" target="_blank">
+        https://cloud.redhat.com.
+      </a>{' '}
+      To discover more managed services, go to the{' '}
+      <Button
+        isInline
+        variant="link"
+        onClick={() => history.push(`/catalog/ns/${currentNamespace}?catalogType=managedservices`)}
+      >
+        managed services catalog.
+      </Button>
+    </Trans>
+  );
+
   return (
-    <FlexForm>
-      <FormBody flexLayout>
-        <FormHeader
-          title={t('rhoas-plugin~Select Kafka Cluster')}
-          helpText={t(
-            'rhoas-plugin~The Kafka cluster selected below will appear on the topology view.',
-          )}
-          marginBottom="lg"
-        />
-        <FormSection fullWidth flexLayout extraMargin>
-          {!areAllServicesSelected(currentKafkaConnections, kafkaArray) ? (
+    <>
+      <FormBody flexLayout style={{ borderTop: 0 }}>
+        <FormSection fullWidth>
+          {connectionError ? (
```

Comment:
[Future] Empty states can be probably extracted into separate component that will return the proper rendering

Reply:
do you mind creating an issue for this to explain your thought process here ?

Reply:
That is fine. some components can be large, but this one is still manageable so no need to do it.